jax/lax/lax.py

```diff
@@ -2922,6 +2922,71 @@ def _pad_translation_rule(c, operand, padding_value, *, padding_config):
 batching.primitive_batchers[pad_p] = _pad_batch_rule
 
 
+# the squeeze primitive exists for the benefit of batching and other rules that
```

Comment:
You might mention why it is preferable as opposed to `reshape`, perhaps with a small example.